ror-tools-core/src/main/scala/tech/beshu/ror/tools/core/utils/RorToolsError.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.tools.core.utils
+
+sealed class RorToolsError(val message: String)
+
+object RorToolsError {
```

Comment:
Those issues are now handled as errors - with error message and exit code 1, but without throwing exception and showing stacktrace. Exceptions are now reserved only for technical/unexpected situations, and not standard behavior, taken into account in logic.